arch/arm/src/tiva/cc13xx/cc13xx_prcm.c

```diff
@@ -1,14 +1,10 @@
 /****************************************************************************
  * arch/arm/src/tiva/cc13xx/cc13xx_prcm.c
  *
- *   Copyright (C) 2018 Gregory Nutt. All rights reserved.
- *   Author: Gregory Nutt <gnutt@nuttx.org>
- *
- * This is a port of TI's prcm.c file (revision 49363) which has a fully
- * compatible BSD license:
```

Comment:
@jerpelea notice that the original text had important information that (i.e. revision number) that with this modification will be lost. Maybe you can include this info after the License header